packages/sanity/src/core/tasks/components/form/tasksFormBuilder/FormEdit.tsx

```diff
@@ -171,6 +173,9 @@ function FormEditInner(props: ObjectInputProps) {
         documentType="tasks.task"
         sortOrder="asc"
         type="task"
+        // TODO: The pane router can't be used in this file, so it's only possibly to access the
+        // globally checked-out bundle. It may be necessary pass the perspective down instead.
+        perspective={router.stickyParams.perspective}
```

Comment:
Given this is for tasks and tasks won't have `perspective` I think we can remove this.

Reply:
Great point. The task documents themselves don't need to have different versions (I think that's what you mean?).

I got my wires crossed here, because we probably do want to allow different version of document to have different tasks. But each of those task documents have only one version.

Reply:
> The task documents themselves don't need to have different versions
Yes! 

This form is to create the task document, and the task document won't have versions, all the ids of the task documents are just published ids.
 
But a version document could have tasks, but that is handled by the id that the task document is referencing, not by the id of the task document.